lib/ansible/modules/net_tools/ldap/_ldap_attr.py

```diff
@@ -36,6 +36,10 @@
     possible to see spurious changes when target and actual values are
     semantically identical but lexically distinct.
 version_added: '2.3'
+deprecated:
+  removed_in: '2.14'
+  why: 'The current "ldap_attr" module does not support LDAP attribute insertations or deletions with objectClass dependencies.'
```

Comment:
```suggestion
  why: 'The current "ldap_attr" module does not support LDAP attribute insertions or deletions with objectClass dependencies.'
```

Reply:
(To be honest, I was'n really sure whether this is correct or not; but since I could find `insertation` in neither Merriam-Webster nor Cambridge Dictionary, I would assume it is not correct.)